frontend/src/App.svelte

```diff
@@ -164,13 +169,18 @@ Answer by running riddle.answer('your answer here') from the console.`);
           {:else}
             <ResourceTreeView
               rootResource="{rootResource}"
+              bind:bottomLeftPane="{bottomLeftPane}"
               bind:resourceNodeDataMap="{resourceNodeDataMap}"
               bind:modifierView="{modifierView}"
             />
           {/if}
         </Pane>
         <Pane slot="second" paddingVertical="{'1em'}">
-          <AttributesView resource="{currentResource}" />
+          {#if bottomLeftPane}
+            <ScriptView bind:bottomLeftPane="{bottomLeftPane}" />
```

Comment:
As per [this discussion](https://github.com/redballoonsecurity/ofrak/pull/265/files/b2dda78d083b1800cb4003bc3011d18b550f60b9#r1147805129) in the preceding comment, this should be changed to the following, with a corresponding change in the `ResourceTreeView` component.

```suggestion
            <svelte:component this="{bottomLeftPane}" />
```